WordPressVIPMinimum/Tests/JS/WindowUnitTest.js

```diff
@@ -0,0 +1,29 @@
+var testString = '?url=www.evil.com';
+var url = "javascript:alert( 'xss' )";
+
+window; // Ok.
+window.location.testing; // Ok.
+window.test = 'http://www.google.com' + testString; // Ok.
+const foo = window.location.testing; // Ok.
+
+var bar = window.location.href; // Warning - variable assignment.
+var c=window.location.hash.split("#")[0]; // Warning - variable assignment.
+
+window.location; // Error.
```

Comment:
How about the bracket notation accessor format e.g

```
window['location'];
window.location['href'];
window['location']['protocol'];
window['location'].host;
```

There should at least be test cases to show these are catered for (and a fix if they currently aren't).